backend/app/api/endpoints/admin.py

```
@@ -1707,3 +1715,643 @@ async def delete_public_retriever(
+        if not bot_name:
+            continue
+
+        # Check if bot exists as a public resource (user_id=0)
+        bot = (
+            db.query(Kind)
+            .filter(
+                Kind.user_id == 0,
+                Kind.kind == "Bot",
+                Kind.name == bot_name,
+                Kind.namespace == bot_namespace,
+                Kind.is_active == True,
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)